integrated_channels/sap_success_factors/client.py

```diff
@@ -216,41 +216,20 @@ def _sync_content_metadata(self, serialized_data):
             ClientError: If SuccessFactors API call fails.
         """
         url = self.enterprise_configuration.sapsf_base_url + self.global_sap_config.course_api_path
-        try:
```

Comment:
we no longer need as much error handling in the clients, the orchestration class handles various error cases. we'll remove more of this redundant code over time